recipes/adam/post-link.py

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Set SPARK_HOME to conda installed pyspark package if not already set
+if [ -z "${SPARK_HOME}" ]; then
```

Comment:
Are these strictly required to be exported?  I would clobber these all the time.

Reply:
pyadam fails if SPARK_HOME and ADAM_HOME are not set.

Reply:
I think what I'm trying to get at is that `pyadam` is not very good.  It could do what `pyspark` does
https://github.com/apache/spark/blob/master/python/pyspark/find_spark_home.py

Reply:
in my experience the pyspark script doesn't really work that well either. It messes up a lot if SPARK_HOME is not set

Reply:
Oh, great.

Reply:
I can't remember the specifics, but there was some circular problem with the pyspark script when I was trying to remove the requirement for SPARK_HOME to be set in mango.